openseeds/cohort-procedures-and-templates/video-facilitator-guide.md

```diff
@@ -0,0 +1,63 @@
+
+We are very excited to create this role in OLS for the returning members from the previous cohorts. 
+
+Our aim is to share our practices for training and planning so you can take them into your projects and community. This is also an opportunity for us to invite community members to take on some of the management roles.
+
+## 💁 Responsibilities
+
+**These facilitators are responsible for the post-call tasks.**
+
+What you need to do: 
+
+- [ ] Please ask the organising team for credentials to access [otter.ai](http://otter.ai/), YouTube and Zoom. We also share this information via Keybase (if you want, you could create a Keybase account, if you don’t have one already).
+- [ ] You can publish the OLS video using your own account. Please create a YouTube account if you don’t have one and ask the Community Coordinator to give you editor access to the OLS YouTube account so you can publish the video. 
+- [ ] Access OLS’ [Shared Drive](https://docs.google.com/presentation/d/1lDLg0xztRfyEgdlJWFDHJMRW6c6k1Yyh02vux8HKDQA/edit#slide=id.g2127687abdb_2_2) to get the slide template for the video thumbnail that you can edit with the cohort call details. 
+- [ ] When the video is published, please share it with the cohort directly via Slack.
+
+#### Please try doing the video processing on the same week of the cohort call, so we can add the link to the recording to the weekly cohort mailing we send every Monday to all cohort members.
+
+_Time commitment: 120-150 minutes per call_
+
+Below we provide a summary checklist. For a detailed tutorial, please refer to [the detailed tutorial](https://docs.google.com/document/d/1c92qDtKLvEM3stKWh0z0uIsqyyLWG37nnsgekbsiiKE/edit#heading=h.hfhzrjvnj7qh).
+
+- [ ] 1. Download from Zoom cloud recordings
+- [ ] 2. Upload to Otter.ai (_will need a few minutes to process_)
+- [ ] 3. Upload to the OLS YouTube account
+- [ ] 4. Create a front page slide for the video - templates are available in Google Drive
+- [ ] 5. Make sure the YouTube video is added to the current cohort playlist
+- [ ] 6. Correct Otter.ai transcripts (_make sure to use the uploaded video, not the original transcript, as the timings won't match the video_)
+- [ ] 7. Download the corrected transcript as a .srt file from Otter
+- [ ] 8. Upload it to the YouTube video
+- [ ] 9. Post a link to the recording in the OLS Slack! 
+
+#### After publishing the video, fill in the following information in our [planning sheet](https://docs.google.com/spreadsheets/d/15cR5YsttvmbtX8q_Zt4YBangzqns3NqkisJFf3_ApZI/edit#gid=1514825681) (the tab is “Nebula 2024-1”)
```

Comment:
We should probably not put the link here directly

Reply:
Are you concerned that someone who shouldn't have access to the document might?

Reply:
What do you recommend? // @bebatut

Reply:
The guide is not specific to a cohort. The sheet is. 
I would replace by something like `in the cohort sheet` (without link and mention of the cohort)